app/views/course/virtual_classrooms/_virtual_classroom.html.slim

```diff
@@ -27,9 +27,10 @@
   div.access-link
     - link_exists = virtual_classroom.instructor_classroom_link.present?
     - is_instructor = can?(:edit, virtual_classroom)
+    - is_main_instructor = virtual_classroom.instructor == current_user
```

Comment:
I'm not sure if you need to check for an instructor, it seems like `is_main_instructor` would be the same value is `link_exists` (correct me if I'm wrong).